core/corehttp/gateway_handler.go

```diff
@@ -352,6 +352,7 @@ func (i *gatewayHandler) getOrHeadHandler(w http.ResponseWriter, r *http.Request
 	// Detect when explicit Accept header or ?format parameter are present
 	responseFormat := customResponseFormat(r)
 	trace.SpanFromContext(r.Context()).SetAttributes(attribute.String("ResponseFormat", responseFormat))
+	trace.SpanFromContext(r.Context()).SetAttributes(attribute.String("ResolvedPath", resolvedPath.String()))
```

Comment:
```suggestion
	trace.SpanFromContext(r.Context()).SetAttributes(
		attribute.String("ResponseFormat", responseFormat),
		attribute.Stringer("ResolvedPath", resolvedPath),
	)
```

Reply:
(using `attribute.Stringer` will avoid allocating and computing the value if tracing's disabled, there's no span, etc.)